docs/releasenotes/README.md

```diff
@@ -1,6 +1,9 @@
 ## Release notes
 
+Major releases and feature releases are indicated in **bold**.
+
 ### Semantic MediaWiki 4.x
+* [SMW 4.1.1 release notes](RELEASE-NOTES-4.1.1.md) (current version)
```

Comment:
Shouldn't this one be bolded and the other "current version" be removed? Or does that happen later?

Reply:
Good catch. I just updated the file.